lib-src/cdd.h

```diff
@@ -35,6 +35,10 @@ extern mytype dd_purezero;
 extern mytype dd_minuszero;
 extern mytype dd_minusone;
```

Comment:
Why did you decide to leave these out of the `extern C` block?

Reply:
just missed that.